### ISSUE TITLE
Nomad status <job> SIGSEV

### ISSUE THREAD

**Issue by jstoja** [hashicorp/nomad#3034]
### Nomad version
`Nomad v0.6.0`

### Operating system and Environment details
`Darwin Juliens-MacBook-Pro.local 15.6.0 Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64 x86_64`
### Issue
I have a cluster of nomad 0.5.6 hosts. I can launch jobs, but I cannot status them.

### Reproduction steps
`nomad status <job_name>`

### Nomad Client logs (if appropriate)
```
➜  ~ nomad status test                              
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4b7f387]

goroutine 1 [running]:
github.com/hashicorp/nomad/command.(*StatusCommand).Run(0xc42043c480, 0xc420010140, 0x1, 0x1, 0xc42014bef0)
	/private/tmp/nomad-20170727-64771-10lwc8x/nomad-0.6.0/src/github.com/hashicorp/nomad/command/status.go:141 +0x927
github.com/hashicorp/nomad/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc420408480, 0xc420408480, 0x29, 0x4eff470)
	/private/tmp/nomad-20170727-64771-10lwc8x/nomad-0.6.0/src/github.com/hashicorp/nomad/vendor/github.com/mitchellh/cli/cli.go:235 +0x2d1
main.RunCustom(0xc420010130, 0x2, 0x2, 0xc4203faf60, 0x0)
	/private/tmp/nomad-20170727-64771-10lwc8x/nomad-0.6.0/src/github.com/hashicorp/nomad/main.go:53 +0xed6
main.Run(0xc420010130, 0x2, 0x2, 0xc4200001a0)
	/private/tmp/nomad-20170727-64771-10lwc8x/nomad-0.6.0/src/github.com/hashicorp/nomad/main.go:23 +0x56
main.main()
	/private/tmp/nomad-20170727-64771-10lwc8x/nomad-0.6.0/src/github.com/hashicorp/nomad/main.go:19 +0x64
```

**Comment by jstoja** [comment]
Since I have many warning with servers not being able to "fetch host disk usage stats" with some deleted alloc directories, it's possible that there is an issue with that.
The weird thing is that even after removing the job and recreating, I get the same issue.

**Comment by jstoja** [comment]
Found that it is fixed in `0.6.1` in the next release with #3033 !
